.storybook/reactIntl.ts

```
@@ -1,11 +1,10 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const boxLanguages = require('@box/languages');
+import boxLanguages from '@box/languages';
 
 const messages = boxLanguages.reduce(
-    (acc, lang) => ({
-        ...acc,
-        // eslint-disable-next-line global-require,import/no-dynamic-require,@typescript-eslint/no-var-requires
-        [lang]: require(`../i18n/bundles/${lang}`).messages, // whatever the relative path to your messages json is
+    (bundles, language) => ({
+        ...bundles,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)